lms/templates/dashboard.html

```diff
@@ -74,10 +74,61 @@
 </div>
 
 <section class="container dashboard" id="dashboard-main">
+  <section class="my-courses" id="my-courses" role="main" aria-label="Content">
+    <header class="wrapper-header-courses">
+      <h2 class="header-courses">${_("Current Courses")}</h2>
+    </header>
+
+    % if len(course_enrollment_pairs) > 0:
+      <ul class="listing-courses">
+      <% dashboard_index = 0 %>
+      % for course, enrollment in course_enrollment_pairs:
```

Comment:
Nitpick: It might be slightly cleaner to use Python's `enumerate` here instead of incrementing an index.